beluga/models.py

```diff
@@ -78,9 +78,18 @@ class Event(Base):
 
     start_time = sa.Column(sa.types.DateTime())
     end_time = sa.Column(sa.types.DateTime())
+    start_time_local = sa.Column(sa.types.DateTime())
+    end_time_loal = sa.Column(sa.types.DateTime())
```

Comment:
typo `loal`->`local`

Reply:
Looks like this is killing the `/events` endpoint

Reply:
True, theres more. I'll tinker with that endpoint a bit ☝️